src/main/webapp/assets/js/controllers/controllers.js

```diff
@@ -258,11 +262,19 @@ canvasSupportApp.controller('addUserController', ['Friend', '$scope', '$rootScop
           } else {
             if(data.data.course_id) {
               $scope.addSuccess = true;
-              var successFullSections = $('#successFullSections').text();
-              $('#successFullSections').text(successFullSections  + ' ' + sectionName);
+              if (checkedSections === sectNumber){
+                $scope.newUser = false;
+                $scope.newUserFound = false;
+                $scope.none = false;
+                $scope.userAvailable  = false;
+                $scope.friendEmailAddress ='';
+                $scope.friend = {};
```

Comment:
So this seems like a reset of all the flags when checkedSections is the same as the section number. I wonder if there will be a need to do this again in the future but somewhere else in the controller. Is this section a good candidate for a small method for reseting flags? Thoughts?